signer/migrations/0003__create_tables.sql

```diff
@@ -184,3 +177,81 @@ CREATE TABLE sbtc_signer.withdrawal_reject_events (
     created_at    TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
 );
 
+-- Represents an individual transaction within a broadcasted sweep transaction
+-- package. Individual deposit and withdrawal requests reference back to these
+-- transactions to keep track of both the overall transaction package as well as
+-- the individual Bitcoin transactions they are related to.
+CREATE TABLE sbtc_signer.sweep_transactions (
+    -- The Bitcoin transaction ID of the transaction.
+    txid BYTEA PRIMARY KEY NOT NULL,
+    -- The signer UTXO being spent in this transaction.
+    signer_prevout_txid BYTEA NOT NULL,
+    signer_prevout_output_index INTEGER NOT NULL,
+    signer_prevout_amount BIGINT NOT NULL,
+    signer_prevout_script_pubkey BYTEA NOT NULL,
+    -- The total amount of the transaction.
+    amount BIGINT NOT NULL,
```

Comment:
Is this the total input amount or the total output amount. Either is fine, just thought that we should specify which.

Reply:
Good point, it's the output amount, will specify in comments.

Reply:
[a0bac87](https://github.com/stacks-network/sbtc/pull/710/commits/a0bac8711f4e83727a72f11d7a147633d8df4bd6)